cargo-verify/src/main.rs

```diff
@@ -438,7 +446,23 @@ fn verifier_run(opt: &Opt, bcfile: &Path, name: &str, entry: &str) -> Status {
         Status::Unknown
     });
 
-    println!("test {} ... {:#}", name, status);
+    if opt.quiet {
+        let s = match status {
+            Status::Unknown => Colour::Yellow.paint("?"),
+            Status::Verified => Colour::Green.paint("."),
+            Status::Error => Colour::Red.paint("F"),
+            Status::AssertFailed => Colour::Red.paint("A"),
+            Status::OutOfBounds => Colour::Red.paint("B"),
+            Status::Overflow => Colour::Red.paint("O"),
+            Status::Panic => Colour::Red.paint("P"),
+            Status::Reachable => Colour::Red.paint("R"),
+            Status::Timeout => Colour::Red.paint("T"),
+        };
+        print!("{}", s);
+    } else {
+        println!("test {} ... {:#}", name, status);
+    }
+    term.flush();
```

Comment:
error[E0425]: cannot find value `term` in this scope

Reply:
Fixed. (I'm not sure if this was a version issue - because I remember seeing that this fixed the problem (of no output until all tests completed)